sone.lua

```diff
@@ -392,8 +393,12 @@ local function biquadFilter(sound, parameters)
     assert(startSample >= 0, "Start time cannot be less than zero")
     assert(finishSample <= sampleCount, "Finish time cannot be longer than the sound")
 
-    for i=startSample, finishSample do
-        sound:setSample(i, clamp(process(sound:getSample(i)), -1, 1))
+    for j = 1, ch do
+        x0, x1, x2 = 0, 0, 0
+        y0, y1, y2 = 0, 0, 0
+        for i=startSample + j - 1, finishSample, ch do
+            sound:setSample(i, clamp(process(sound:getSample(i)), -1, 1))
+        end
```

Comment:
Not sure if Love ever plans to support more than 2 channels; probably doesn't need to be a loop. I went for future-proofing it over saving a few cycles, but feel free to change it of course.